src/client/api/events/sub_milestone_payload.c

```diff
@@ -21,7 +26,7 @@ int parse_milestone_latest(char *data, milestone_latest_t *res) {
   }
 
   // Parse timestamp
-  if ((json_get_uint64(json_obj, JSON_KEY_TIMESTAMP, &(res->timestamp))) != 0) {
+  if ((json_get_uint32(json_obj, JSON_KEY_TIMESTAMP, &(res->timestamp))) != 0) {
```

Comment:
One parentheses too many.
```suggestion
  if (json_get_uint32(json_obj, JSON_KEY_TIMESTAMP, &(res->timestamp)) != 0) {
```